From 14d61c97d5a3cd91dc04d3fc5c3497b935403aaa Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Mon, 01 Aug 2022 16:40:06 +0300
Subject: [PATCH] [SW-0] erom: spare functiona call

erom_main_function() function does nothing but call to erom_tpm()
function. Spare this redundant call.

Change-Id: I905e8c1d127f74d547f0b68d4e45c14e8f460130
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/app/ext_rom/erom_tpm.c b/app/ext_rom/erom_tpm.c
index c18d589..e5dea28 100644
--- a/app/ext_rom/erom_tpm.c
+++ b/app/ext_rom/erom_tpm.c
@@ -127,7 +127,7 @@
 
 __weak void ppboot_secure_arc(void) {}
 
-void erom_tpm(void)
+void erom_main_function(void)
 {
 	uint32_t offset = 0;
 	uint16_t size = 0;
@@ -153,9 +153,4 @@
 
 	if (tpm_hash_and_pcr_extend(TPM2_PPBOOT_PCR_IDX, offset, size))
 		return;
-}
-
-void erom_main_function(void)
-{
-	erom_tpm();
 }
